packages/plugins/ScamSniffer/src/SiteAdaptor/index.tsx

```diff
@@ -90,10 +91,11 @@ const site: Plugin.SiteAdaptor.Definition = {
                 matchType: 'sim',
             } satisfies ScamResult
         }, [firstHit?.host, firstHit?.url, author?.userId, nickname])
-
-        if (!isTwitter) return null
         const project = scamProject || fallbackScamProject
-        if (!project) return null
+
+        usePluginWrapper(!!project)
```

Comment:
```suggestion
        usePluginWrapper(!!(project && isTwitter))
```

Reply:
This cannot detect if the `project` type is not null.